src/main/java/uk/gov/hmcts/darts/audio/service/impl/AdminMediaServiceImpl.java

```diff
@@ -123,4 +131,29 @@ private List<HearingEntity> getApplicableMediaHearings(MediaEntity mediaEntity,
 
         return hearingEntityList;
     }
+
+    @Override
+    @Transactional
+    public MediaApproveMarkedForDeletionResponse adminApproveMediaMarkedForDeletion(Integer mediaId) {
+        MediaApproveMarkedForDeletionResponse response;
+
+        mediaApproveMarkForDeletionValidator.validate(mediaId);
+        List<ObjectAdminActionEntity> objectAdminActionEntityList = objectAdminActionRepository.findByMedia_Id(mediaId);
+
+        Optional<MediaEntity> mediaEntityOptional = mediaRepository.findById(mediaId);
+        if (mediaEntityOptional.isPresent()) {
```

Comment:
consider inverting IF statement and having the happy path outwith of the if statement, see other comment.